massa-protocol-worker-2/src/connectivity.rs

```diff
@@ -189,15 +197,19 @@ pub fn start_connectivity_thread(
                                 {
                                     {
                                         let active_connections = manager.active_connections.read();
-                                        println!("Checking addr: {:?}", addr);
-                                        println!("Connections queue = {:#?}", active_connections.connection_queue);
-                                        println!("Connections = {:#?}", active_connections.connections);
+                                        if config.debug {
+                                            println!("Checking addr: {:?}", addr);
```

Comment:
if u use debug! instead of println you can remove the bool ?

Reply:
the boolean is there to avoid having the print only on one of the two managers in the tests and make ti more readable